src/providers/Celo.ts

```diff
@@ -211,6 +214,27 @@ export async function getCurveUSDC(): Promise<ProviderResult> {
   }
 }
 
+export async function getUniV3Holdings(
+  address: string
+): Promise<ProviderResult<Map<string, number>>> {
+  try {
+    const uniV3Holdings = await uniV3BalanceCalculator.calculateUniV3PoolBalance(address)
+    if (uniV3Holdings.has(STAKED_CELO_ERC20_ADDRESS)) {
+      uniV3Holdings.set(
+        STAKED_CELO_ERC20_ADDRESS,
+        (
+          await StakedCeloProvider.Instance.stCeloToCelo(
+            new BigNumber(uniV3Holdings.get(STAKED_CELO_ERC20_ADDRESS))
+          )
+        ).toNumber()
+      )
+    }
+    return providerOk(uniV3Holdings, Providers.celoNode)
+  } catch (error) {
+    return providerError(error, Providers.celoNode)
+  }
```

Comment:
I wouldn't put this logic here, but do it down the line when accumulating all celo, balance. I.e. add the actual celo, and also add the converted staked celo from there and keep the holdings as purely the token holdings from UniV3 pools, otherwise things might get weird.

Reply:
Yeah I agree I also didn't like doing this here but further down i would need to initiate the StakedCeloProvider again in holdings.ts. Is that what you suggest?

Reply:
Yeah given that it's a singleton Instance there's no problem accessing it from everywhere